x-pack/legacy/plugins/infra/public/pages/metrics/components/section.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FunctionComponent, useCallback } from 'react';
+import { EuiTitle } from '@elastic/eui';
+import { SideNavContext, SubNavItem } from '../lib/side_nav_context';
+import { LayoutProps } from '../types';
+
+type SectionProps = LayoutProps & {
+  navLabel: string;
+  sectionLabel: string;
+};
+
+export const Section: FunctionComponent<SectionProps> = ({
```

Comment:
Just out of curiosity, why do we have `FunctionComponent<SectionProps>`? Why not just:

```ts
export const section = (props: SectionProps)
```

Reply:
`FunctionComponent` will add the `children` prop (correctly) to the props definition.